plugins/jdg70/src/main/resources/server.xml

```diff
@@ -0,0 +1,399 @@
+<?xml version="1.0" ?>
+
+<server xmlns="urn:jboss:domain:4.0">
```

Comment:
I just copied the file from the ER2 build, so it could change. I'm not sure what you mean by symlinking and keeping the modifications in RG sources. You mean doing XML transformations in RG?